packages/kbn-es-query/src/filters/index.d.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import {
+  ExistsFilter,
+  PhraseFilter,
+  PhrasesFilter,
+  QueryFilter,
+  RangeFilter,
+} from 'ui/filter_bar/filters';
```

Comment:
It looks like there are very few references to the `ui/public` folder in these packages (aside from these new references, I see only one in `kbn-plugin-generator`.  Would you mind giving me a high level of what is `ui/public/filter_bar`, `ui/public/search_bar`, `kbn-es-query` ?  It looks like there is code in `ui/public/filter_bar` that imports `FilterOptions` from `ui/public/search_bar`?  I'm just not sure I follow the organization of all of this now, and also where it would be in an ideal situation (thinking back to the discussion we had on moving everything out of `ui/public` - where would these things go?)

Reply:
@Stacey-Gammon  Sorry for the confusion, one of our few remaining tasks is to reorganize files and delete a bunch of stuff. This package ultimately won't import from ui/public at all. I'll ping you when that's all done and provide an explanation of what lives where.

Reply:
P.S. This has been done @Stacey-Gammon. The filter objects are now a part of the kbn-es-query package instead of the filter bar library, so these imports have been updated.